### ISSUE TITLE
Add Service Hub models

### ISSUE THREAD

**Issue by jamesrayoub** [fivetran/dbt_hubspot_source#27]
Fivetran added the Service Hub tables to the connector last year and the following staging models should be created

- `ticket`
- `ticket_company`
- `ticket_contact`
- `ticket_deal`
- `ticket_engagement`
- `ticket_property_history`

**Comment by jamesrayoub** [comment]
opened #28 to address this

**Comment by fivetran-joemarkiewicz** [comment]
Hi @jamesrayoub PR #30 implements the Service Hub changes you initiated within PR #28. We are holding off merging the PR to master until the additional tables you requested with the [Support Portal](https://support.fivetran.com/hc/en-us/community/posts/360053108713-Hubspot-Tickets-Stage-Labels) are added at the connector level. This way we may fully capture the Service Hub models.

**Comment by jamesrayoub** [comment]
Great, thanks for the update @fivetran-joemarkiewicz

**Comment by fivetran-joemarkiewicz** [comment]
Hi James, I was able to add the `ticket_pipeline` and `ticket_pipeline_stage` tables to service-hub-updates branch!

Additionally, in response to your PR #35 I leveraged the code you provided and integrated it with how we have handled pass through columns in the past. The modifications I made should hopefully make declaring the variable a bit easier and will be consistent with our previous package pass through variables. 

The approach I took was to only provide the opinionated "standard" fields for the `company`, `contact`, `deal`, and `ticket` within the respective macros and staging models, then use a conditional break to call the variables the customer will set to pass them through to the final staging model.

Let me know if you have any questions on this approach. Also, feel free to install this working branch as a dependency prior to us merging to master and releasing a new version as I would greatly appreciate your feedback and contributions beforehand!
```
packages:
  - git: https://github.com/fivetran/dbt_hubspot_source.git
    revision: service-hub-updates
    warn-unpinned: false
```
